components/modules/utils/config/audioPlayer.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState, useEffect, useRef, FC } from "react";
+import AudioControls from "./audioControls";
+
+type TrackItems={
+  tracks: any;
+  [index: number]:{title: string;
```

Comment:
You missed the closing curly brace, and again we will need an explanation about this notion on the index

```suggestion
  [index: number]:{title: string;}
```